BlakeDonn/rancid-tomatillos
# Login

### What's this PR do?  
- Adds functionality to the login page 
- Login page now uses state to capture user input for email and password
- Login page now sends POST request when login button is clicked
- Login page now correctly redirects to mainview upon successful login 
- Upon successful login the loggedIn state updates on the app state to be used to render a personalized view

### Where should the reviewer start?  
- Check the new props being passed to the login component (includes Route props and callback function)

### How should this be manually tested?  
- Open the app and test sad and happy logins, check the react dev tools to make sure state is correctly updated on redirect

### Any background context you want to provide?  
- Got the Route props and the callback function to both pass to the Login component!

### What are the relevant tickets?  
closes #2 closes #38 closes #37 closes #24 closes #23 closes #19 closes #29 closes #6 closes #36
